website/templates/emails/pending_registration_admin.txt.mako

```diff
@@ -0,0 +1,16 @@
+Hello ${user.fullname},
+
+
+We just wanted to let you know that ${initiated_by} has initiated the following pending registration: ${registration_link}
+
+To approve this action click the following link: ${approval_link}
+
+To disapprove this action, click the following link: ${disapproval_link}
+
+Note: Clicking the disapproval link will immediately cancel the pending registration. If you neither approve nor disapprove
+the registration within ${approval_time_span} hours of midnight tonight (EDT) the registration will become automatically
```

Comment:
This has a few extra edits, but the most important is that I agree we should take out the midnight part. I'd say:

Hello _____,

_____ has initiated a registration of your project ________. The pending registration can be viewed here: _____.

To approve this registration, click the following link: ______

To immediately cancel this registration, click the following link: ________

Note: If you take no action within _____ hours, the registration will be automatically approved. This operation is irreversible.

Sincerely yours,

Whatever we sign this.